config/locales/views/en.yml

```diff
@@ -952,4 +956,4 @@ en:
         revealed_html: A translation of %{work_link} by %{creator_link}
         unrevealed: A translation of a work in an unrevealed collection
     work_module:
-      draft_deletion_notice_html: This draft will be <strong>scheduled for deletion</strong> on %{deletion_date}.
+      draft_deletion_notice_html: This draft will be <strong>scheduled for deletion</strong> on %{deletion_date}
```

Comment:
Just now noticing this... I ran `docker compose exec web bundle exec i18n-tasks normalize -l en #` and this probably happened during that. I can remove this change if you'd like.

Reply:
That's strange. I think you should remove the change, i18n-tasks should hopefully still accept the file as normalized.

Reply:
Great, I added the period back in.